README.md

```diff
@@ -210,6 +210,41 @@ Valid patch directories include:
 
 **Nota Bene:** You need to be running at least Bash v4 to use this command.
 
+### `e exec`
+
+Returns the path to the current configuration's Electron executable.
+Useful in combination other shell tools.
+
+Example Usage:
+
+```sh
+$ uname
+Linux
+$ e exec
+/home/username/electron-gn-root/src/out/Testing/electron
+
+$ uname
+Darwin
+$ e exec
+/Users/username/electron-gn-root/src/out/Testing/Electron.app/Contents/MacOS/Electron
+```
+
+### `e src`
```

Comment:
maybe we could consolidate these PRs under one umbrella, since they both serve a similar purpose?

perhaps
```
e show [path]
```

Reply:
Mmmaybe? If there are more use cases for an `e show` than just exec + src, I could see that as a logical grouping.

If in practice we only have the two cases, I've got a slight preference to keep them as-is because `e show src base` is borderline prolix.

Reply:
Just as a third one I had in an old script `e show out` to get the path to the current out directory.  Used in an old alias to run `mksnapshot` (`${egetout}/mksnapshot`).  Not set either way just noting a third potential path people may want

Reply:
Added an umbrella subcommand `get` and the outdir option in d08d9e8b0474bd870d9c60082ee36886308e2262